src/dmd/cli.d

```diff
@@ -73,6 +73,24 @@ struct Option
     string flag; /// The CLI flag without leading `-`, e.g. `color`
     string helpText; /// A detailed description of the flag
     TargetOS os = TargetOS.all; /// For which `TargetOS` the flags are applicable
+    string ddocText; /// Detailed description of the flag (in Ddoc)
+
+    ///
+    this(string flag, string helpText, TargetOS os = TargetOS.all)
```

Comment:
Ideally all these parameters should be documented.

Reply:
Done.